.github/workflows/advanced.yml

```diff
@@ -205,15 +245,15 @@ jobs:
         run: |
           .github/scripts/build-openblas.sh
 
-      - name: Pack OpenBLAS
+      - name: Pack OpenBLAS tests
```

Comment:
I would add caching to every package anyway, it does not take too much space and can be beneficial in some cases.

Reply:
This needs to be though through thoroughly. Caching is powerful but dangerous thing. For example, the cache key of OpenBLAS buld whould have to consist of the current toolchain three component key, the OpenBLAS revision, and some sort of hash computed from OpenBLAS build scripts to work properly.